api/src/paths/project/{projectId}/update.ts

```diff
@@ -302,6 +312,24 @@ export const getSpeciesData = async (projectId: number, connection: IDBConnectio
   return new GetSpeciesData(resultFocalSpecies, resultAncillarySpecies);
 };
 
+export const getObjectivesData = async (projectId: number, connection: IDBConnection): Promise<GetObjectivesData> => {
+  const sqlStatement = getObjectivesByProjectSQL(projectId);
+
+  if (!sqlStatement) {
+    throw new HTTP400('Failed to build SQL statement');
+  }
+
+  const response = await connection.query(sqlStatement.text, sqlStatement.values);
+
+  const result = (response && response.rows && response.rows[0]) || null;
+
+  if (!result) {
+    throw new HTTP400('Failed to get project coordinator data');
```

Comment:
wrong error message here, should probably say:

```suggestion
    throw new HTTP400('Failed to get project objectives data');
```